bedrock/settings/base.py

```diff
@@ -1181,6 +1181,7 @@ def lazy_email_backend():
 
 # country code for /country-code.json to return in dev mode
 DEV_GEO_COUNTRY_CODE = config('DEV_GEO_COUNTRY_CODE', default='US')
+SEND_TO_DEVICE_COUNTIRES = config('SEND_TO_DEVICE_COUNTIRES', default='US', cast=Csv())
```

Comment:
`s/COUNTIRES/COUNTRIES` 😉 

(Also change in `helpers.py`)

Reply:
LOL! Great catch 😆